lte/gateway/docker/mme/Dockerfile.rhel8

```diff
@@ -138,7 +138,7 @@ RUN  git clone --recurse-submodules -b v1.15.0 https://github.com/grpc/grpc && \
      wget https://ftp.gnu.org/gnu/nettle/nettle-2.5.tar.gz && \
      wget https://www.gnupg.org/ftp/gcrypt/gnutls/v3.1/gnutls-3.1.23.tar.xz && \
      git clone https://liblfds.org/git/liblfds && \
-     git clone https://git.osmocom.org/libgtpnl && \
+     git clone https://gitea.osmocom.org/cellular-infrastructure/libgtpnl/ && \
```

Comment:
nit³: the "/" at the end is superfluous, right? -> inconsistent with the other entries